app/assets/stylesheets/scaffolds.scss

```
@@ -65,6 +65,10 @@ body {
   font-size: 18px;
 }
 
+a {
+  text-decoration: none;
+}
+
```

I mean to take out the `:hover` only. This is what should have remained.

```suggestion
a {	
  color: var(--link-branded-color);	
  text-decoration: none;
}
```